CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v6.0.1 (2025-02-14)
+
+## OS Changes
+* Update `containerd` from 1.7.24 to 1.7.25 ([#374])
+* Add patches to `containerd` 1.7.25 ([#374])
```

Comment:
I'd prefer to see a bit more information about the intent of the patches here.

Reply:
Updated.